.github/workflows/database-copy.yml

```diff
@@ -0,0 +1,18 @@
+name: Copy PAAS databases to AKS
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 2 * * *" # 02:00 UTC
+
+jobs:
+  restore_db:
+    name: Restore DB
+    strategy:
+      matrix:
+        environment: [dev]
+      max-parallel: 1
+    uses: ./.github/workflows/restore-paas-db-to-aks.yml
```

Comment:
believe that means we need workflow_call added to restore-paas-db-to-aks.yml
i.e.
on:
  workflow_call:
    inputs:
      environment:
        type: string